src/main/java/me/notlewx/gamemode/Main.java

```diff
@@ -9,16 +11,21 @@ public final class Main extends JavaPlugin {
 
     @Override
     public void onEnable() {
-        if (Bukkit.getPluginManager().getPlugin("BedWars1058") == null) {
-            getLogger().severe("BedWars1058 was not found. Disabling...");
-            Bukkit.getPluginManager().disablePlugin(this);
+        if (!Bukkit.getPluginManager().isPluginEnabled("BedWars1058") && !Bukkit.getPluginManager().isPluginEnabled("BedWars2023") ) {
+            getLogger().severe("BedWars1058 or BedWars2023  was not found. Disabling...");
+            setEnabled(false);
             return;
         }
-        else {
+        if (Bukkit.getPluginManager().isPluginEnabled("BedWars1058")){
             System.out.println(ChatColor.translateAlternateColorCodes('&', "[BedWars1058-AdventureMode] &aBedWars1058 found... Hooking!"));
             System.out.println(ChatColor.translateAlternateColorCodes('&', "[BedWars1058-AdventureMode] &aRunning on" + "&b" + getServer().getVersion()));
+            getServer().getPluginManager().registerEvents(new ArenaListenerBW1058(), this);
+        } else if (Bukkit.getPluginManager().isPluginEnabled("BedWars2023")){
+            System.out.println(ChatColor.translateAlternateColorCodes('&', "[BedWars1058-AdventureMode] &aBedWars2023 found... Hooking!"));
+            System.out.println(ChatColor.translateAlternateColorCodes('&', "[BedWars1058-AdventureMode] &aRunning on" + "&b" + getServer().getVersion()));
```

Comment:
Do you mean changing this to BedWars2023-AdventureMode?
Btw this should be changed to use the logger :)

Reply:
Yes, exactly that.

Wdym, what should be changed

Reply:
sys out should not be used in plugins. its should use the logger instead

Reply:
Ah, i understood. tbh since its a really tiny addon i didnt care much about it, but yeah we can add if you wish